src/main/resources/application.properties

```diff
@@ -11,4 +11,6 @@ management.server.port=8081
 management.endpoints.enabled-by-default=false
 management.endpoint.health.enabled=true
 management.endpoint.prometheus.enabled=true
-management.endpoints.web.exposure.include=prometheus,health
\ No newline at end of file
+management.endpoints.web.exposure.include=prometheus,health
+
+usecryptoService = false
```

Comment:
For å tydeliggjøre at dette er en feature toggle, skal vi døpe den om til typ `featureToggle.useCryptoService`?